README.md

```diff
@@ -1 +1,92 @@
-Easypay Checkout SDK
+# Easypay Checkout SDK
+
+Easypay's Checkout SDK is intended to facilitate integration between your application and our solution to receive payments.
+
+![easypay-checkout](https://user-images.githubusercontent.com/30448483/171024493-dbda5a46-45d9-4810-ba89-a276d88c770b.gif)
+
+## Installation:
+
+Install the package with:
+
+```sh
+npm install --save @easypaypt/checkout-sdk
+
+# or
+
+yarn add @easypaypt/checkout-sdk
+```
+
+## Usage:
+
+### Import
+
+```js
+import { startCheckout } from "@easypaypt/checkout-sdk"
+
+const checkoutInstance = startCheckout(manifest, [options])
+```
+
+**`manifest`** - server to server response from checkout session creation
+
+| Option      | Type       | Required | Default                  | Description                                                                                  |
+| ----------- | ---------- | -------- | ------------------------ | -------------------------------------------------------------------------------------------- |
+| `id`        | `string`   | false    | `checkout-form`          | parameter to link the element in the page                                                    |
```

Comment:
Will talk in person about omitting the Required column and how to explain the default message handler.